core/docker/Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN \
     chown -R "trino:trino" /usr/lib/trino /data/trino
 
 ARG TRINO_VERSION
-COPY trino-cli-${TRINO_VERSION}-executable.jar /usr/bin/trino
+COPY --chown=trino:trino trino-cli-${TRINO_VERSION}-executable.jar /usr/bin/trino
```

Comment:
This is the only significant change. Previously the file was copied as root with broader access then needed

Reply:
separate preparatory commit?